lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -2152,136 +2164,195 @@ def test_courses_data_time_series_metrics(self):
                         StudentGradebook.objects.create(user=user, course_id=course.id, grade=0.9,
                                                         proforma_grade=0.91)
 
-        # make more completions
+        # Submit scores for the second module for the first five users
+        # Pretend the scores were submitted over the course of the final five days
         for j, user in enumerate(users[:5]):
-            complete_time = timezone.now() + relativedelta(days=-(USER_COUNT - j))
-            with freeze_time(complete_time):
-                module = self.get_module_for_user(user, course, item2)
+            submit_time = datetime.now() - timedelta(days=(5 - j))
+            with freeze_time(submit_time):
                 grade_dict['user_id'] = user.id
-                module.system.publish(module, 'grade', grade_dict)
+                second_module = self.get_module_for_user(user, course, item2)
+                second_module.system.publish(second_module, 'grade', grade_dict)
 
-        test_course_id = unicode(course.id)
-        # get course metrics in time series format
-        end_date = datetime.now().date()
-        start_date = end_date + relativedelta(days=-4)
-        course_metrics_uri = '{}/{}/time-series-metrics/?start_date={}&end_date={}&organization={}'\
-            .format(self.base_courses_uri,
-                    test_course_id,
-                    start_date,
-                    end_date,
-                    org_id)
+        ### OKAY, WE"VE FINISHED THE SETUP WORK FOR THIS INDIVIDUAL TEST CASE ###
+
+        # Generate the time series report for the first five days of the set, filtered by organization
+        # There should be one time series entry per day for each category, each day having varying counts
+        end_date = datetime.now() - timedelta(days=(USER_COUNT - 4))
+        start_date = datetime.now() - timedelta(days=USER_COUNT)
+        course_metrics_uri = '{}/{}/time-series-metrics/?start_date={}&end_date={}&organization={}'.format(
+            self.base_courses_uri,
+            unicode(course.id),
+            start_date,
+            end_date,
+            org_id
+        )
         response = self.do_get(course_metrics_uri)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.data['users_not_started']), 5)
         total_not_started = sum([not_started[1] for not_started in response.data['users_not_started']])
-        self.assertEqual(total_not_started, 6)
+        self.assertEqual(total_not_started, 105)  # 105 aggregate total in the first five days (25, 24, 23, 22, 21)
```

Comment:
The sum of those five numbers is 115, not 105.  I believe they should be 24, 23, 22, 21, 20, which would add up to 110.  Not sure where the 105 comes from.